easybuild/easyblocks/s/scotch.py

```diff
@@ -108,7 +117,7 @@ def build_step(self):
         if self.toolchain.options['i8']:
             cflags += " -DINTSIZE64"
 
-        if not self.toolchain.mpi_family() in [toolchain.INTELMPI, toolchain.QLOGICMPI]:  #@UndefinedVariable
+        if self.cfg['threadedmpi']: 
```

Comment:
How do I check a variable has been set in the easyconfig? I need to know if they have explicitly set the value to false in the easyconfig so I can ensure I only change the value to true when it's not defined there.